docs/specification/1.0.md

```diff
@@ -523,6 +524,36 @@ The following example illustrates a delete action:
 }
 ```
 
+
+
+#### overrideReason
+
+An **overrideReason** is described by the following attributes. No standard set of **overridereasons** exists today; implementers are encouraged to submit suggestions for standardization. To enable analysis of a card's usage, the CDS service SHALL send the same `code`, `codesystem`, and `label` across CDS Hooks requests for the same CDS client.
+
+Field | Optionality | Type | Description
+----- | ----- | ----- | --------
+`code` | REQUIRED | *string* | A code for this **overridereason**. `code` MUST be unique within the card.  To enable analysis of a card's usage, the CDS service SHALL send the same `code` accompanied by the same `label` across CDS Hooks requests for the same CDS client.
```

Comment:
Account for system in this text: "SHALL send the same `code` and `system` accompanied by the same `label` across CDS..."